tests/test_gui.py

```diff
@@ -116,11 +116,16 @@ def test_my_tasks(app_dialog, tk_test_project, tk_test_entities, tk_test_current
     My Tasks tab validation
     """
     # Wait for the UI to show up, click on the home button and make sure My Tasks tab is selected by default
-    app_dialog.root.buttons["Click to go to your work area"].waitExist(timeout=30)
-    app_dialog.root.buttons["Click to go to your work area"].mouseClick()
-    assert app_dialog.root.tabs[
-        "My Tasks"
-    ].selected, "My Tasks tab should be selected by default"
+    app_dialog.root.tabs["Activity"].waitExist(timeout=30)
```

Comment:
maybe this part should be modified by this app_dialog.root.buttons["Click to go to your work area"].waitExists(timeout=30)